src/main/java/io/cdap/e2e/pages/locators/CdfStudioLocators.java

```diff
@@ -180,7 +180,8 @@ public static WebElement locatePluginGroupCollapsed(String pluginGroupName) {
 
   public static WebElement locatePluginNameInList(String pluginName, String  pluginGroupName) {
     String xpath = "//div[@data-cy='plugin-" + pluginGroupName + "-group']" +
-      "//div[contains(@class, 'plugin-name')][normalize-space(text()) = '" + pluginName + "']";
+      "//div[contains(@class, 'PluginNameContainer')]" + 
+      "[translate(normalize-space(text()), ' ', '') = '" + pluginName + "']";
```

Comment:
`normalize-space` only removes leading/trailing spaces. Not sure how it was working before. I am using translate function to remove spaces inside the text as well. I could remove `normalize space` but I guess having it there doesn't hurt.